core/src/main/java/org/mule/runtime/core/internal/streaming/ManagedCursorProvider.java

```diff
@@ -19,14 +26,23 @@
  */
 public abstract class ManagedCursorProvider<T extends Cursor> implements CursorProvider<T> {
 
+  private static final Logger LOGGER = getLogger(ManagedCursorProvider.class);
+
   private final CursorProvider<T> delegate;
   private final CursorManager cursorManager;
-  private final CursorContext cursorContext;
+  private final CursorContext cursorContext; //TODO: Seems no longer necessary
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS1135))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7643&issues=AWirZZm9otIXnmsUDp_S&open=AWirZZm9otIXnmsUDp_S)

[ISSUE_KEY]: AWirZZm9otIXnmsUDp_S